ArithmeticServer.ts

```diff
@@ -18,9 +18,9 @@ http.createServer(function (request, response) {
             ans = 'num must be used with fibonacci pathname!';
         }
         else {
-            n = parseInt(queryData.num);
+            let n:number = parseInt(queryData.num);
```

Comment:
Please extract `parseInt` to a function to better encapsulate how to parse ints.